### DISCUSSION TITLE
FR: Custom row title background-color

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#38310] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Please, add custom row title background-color

**Why is this needed**:

I'd like to separate (by color) between repeated (templating) rows, by default with transparent color the separation is barely noticeable

![image](https://user-images.githubusercontent.com/2898361/122752759-26640480-d2b3-11eb-856f-e2b681186573.png)

It's possible to separate by Stat panel but appears extra spaces 

![image](https://user-images.githubusercontent.com/2898361/122753287-da658f80-d2b3-11eb-803d-c6798bea0f8c.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
